modeling/data.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package modeling
+
+import "errors"
+
+// Zone mock struct for real zones in k8s
+type Zone struct {
+	// Number of nodes of this zone
+	Nodes int
+	// Number of endpoitns of this zone
+	Endpoints int
+	// Name of this zone
+	Name string
+	// Number of pods / total number of pods of all zones
+	endpointsRatio float64
+	// Number of nodes / total number of nodes of all zones
+	nodesRatio float64
+}
+
+type zoneInfos struct {
```

Comment:
I think it might not be perfectly accurate to refer all the zones as a cluster. Can we just keep them as zones since cluster might bring in a new 'topology' relation.

Reply:
renamed to regionInfo